websites/G/G2G/presence.ts

```diff
@@ -0,0 +1,97 @@
+const presence = new Presence({
+		clientId: "1073167802296438875",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/XkzTOsp.png",
+		startTimestamp: browsingTimestamp,
+	};
+
+	switch (document.location.pathname.split("/")[1]) {
```

Comment:
Properties of `document.location` are used many times. Consider destructuring them:
```ts
const { pathname, href, search } = document.location;
```